tenablesc/report.go

```diff
@@ -0,0 +1,86 @@
+package tenablesc
+
+import (
+	"fmt"
+)
+
+const reportEndpoint = "/report"
+
+// Report represents request/response structure from https://docs.tenable.com/tenablesc/api/Report.htm
+type Report struct {
+	BaseInfo
+	ReportDefinitionID string `json:"reportDefinitionID"`
+	JobID              string `json:"jobID"`
+	Type               string `json:"type"`
+	Status             string `json:"status"`
+	Running            string `json:"running"`
+	ErrorDetails       string `json:"errorDetails"`
+	TotalSteps         string `json:"totalSteps"`
+	CompletedSteps     string `json:"completedSteps"`
+	StartTime          string `json:"startTime"`
+	FinishTime         string `json:"finishTime"`
+	OwnerGID           string `json:"ownerGID"`
+	PubSites           []struct {
+		BaseInfo
+	} `json:"pubSites"`
+	Creator struct {
+		ID        string `json:"id"`
+		Username  string `json:"username"`
+		Firstname string `json:"firstname"`
+		Lastname  string `json:"lastname"`
+		UUID      string `json:"uuid"`
+	} `json:"creator"`
+	Owner struct {
```

Comment:
Could this be `UserInfo`?

Reply:
Addressed in https://github.com/palantir/tenablesc-client/pull/28/commits/3a936f275c81fe1a175e00f915e76a11fce296ce